.github/workflows/gradle.yml

```diff
@@ -20,6 +20,12 @@ jobs:
 
     steps:
       - uses: actions/checkout@master
+      - name: Check limited Guava usage
+        run: |
+          if grep -q -r --include \*.java com.google.common.base.Objects; then
```

Comment:
1. We should  do `grep -r com.google.common.base.Objects .` in order to see affected files. I don't know why `--include` is needed.
2. This check should run only on `ubuntu`

Reply:
> 1. We should  do `grep -r com.google.common.base.Objects .` in order to see affected files. I don't know why `--include` is needed.

OK, I didn't know if we want any output of this grep or not.
Well, to skip files other than .java, so this is not needed, but I think this do no charm. I you want, I will remove this.

> 2. This check should run only on `ubuntu`

OK.

Reply:
I've added ``--include`` so i.e. if someone will add this in some docs or README there will be no error.

Reply:
> I've added `--include` so i.e. if someone will add this in some docs or README there will be no error.

This PR is meant to not use Guava's `Objects` class including code and docs. I see no meaning to refer Guava's `Objects` class in docs. Applying the check to entire project is more meaningful.

Reply:
Well, as you wish. On my local machine grep always finds gradle.yml, but if it's not a case than OK. I'll change it exactly as you wrote it.

Reply:
You are right. We would need to exclude `.github` directory to skip `.github/workflow/gradle.yml` file not to be matched. I tried in my local clone with `grep -r --exclude-dir=.github com.google.common.base.Objects .` worked as expected.